imap_processing/ultra/l1a/ultra_l1a.py

```diff
@@ -0,0 +1,113 @@
+"""Contains code to perform ULTRA L1a science processing."""
+import dataclasses
+import logging
+from pathlib import Path
+
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import calc_start_time, write_cdf
+from imap_processing.ultra import ultra_cdf_attrs
+from imap_processing.ultra.l0.decom_ultra import ULTRA_AUX, decom_ultra_apids
+
+logger = logging.getLogger(__name__)
+
+
+def xarray_aux(decom_ultra_aux: dict):
+    """Create xarray for auxiliary packet.
+
+    Parameters
+    ----------
+    decom_ultra_aux : dict
+        Parsed data.
+
+    Returns
+    -------
+    dataset : xr.Dataset
```

Comment:
This will fail in document. I didn't checked in your last PR but you may want to run document command locally to check if your new code is showing up on document page as you like.
```suggestion
    dataset : xarray.Dataset
```